### DISCUSSION TITLE
Lock feature for videos

### DISCUSSION THREAD

**Discussion by ndrwbrns** [gopro/labs#1147] [Feature Requests]
Hi, would it be possible to add a lock feature for videos so that they don't get accidentally deleted from the micro sd card? I use my gopro as a dash cam and almost had a near miss today and wanted to make sure that the video was secured, but then realized that there is no lock feature... would be a nice and useful addition! :).

**Comment by dnewman-gpsw** [comment]
It might not be possible to lock particular files. However, if you use alternative filenaming, you can't accidentally delete them on camera (other than formatting.)  https://gopro.github.io/labs/control/basename/  However, is it so successful, looping mode will not delete the older  clips -- so don't using with Looping.
